bundles/org.openhab.binding.homewizard/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="homewizard:p1_wifi_meter">
```

Comment:
Some new channels seem missing, can you check all?

Reply:
Yes, added the missing ones.